tests/test_routes.py

```diff
@@ -245,6 +244,27 @@ def test_first_rating_product(self):
         new_product = response.get_json()
         self.assertAlmostEqual(new_product["rating"], 3)
 
+    def test_update_price(self):
+        '''It should update the price of a product'''
+        # create a product to update
+        test_product = ProductFactory()
+        response = self.client.post(BASE_URL, json=test_product.serialize())
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        id = response.get_json()["id"]
+
+        # update the product price
+        new_product = {}
+        new_product["price"] = int(MAX_PRICE)
+        response = self.client.put(f"{BASE_URL}/{id}/price", json=new_product)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        updated_product = response.get_json()
+        self.assertEqual(updated_product["price"], float(MAX_PRICE))
```

Comment:
If possible, I think it is better to use `self.assertAlmostEqual` as two `float` datatypes cannot be compared for `equality`